portfolio/src/main/webapp/script.js

```diff
@@ -64,3 +65,19 @@ async function getMessages() {
 
   document.querySelector('#messages').innerHTML = messages;
 }
+
+async function checkLogin() {
+  const response = await fetch('/login');
+  const data = await response.json();
+
+  var loginLink = document.querySelector('#login');
+
+  if (data.isLoggedIn) {
+    loginLink.innerText = 'Logout';
+  } else {
+    loginLink.innerText = 'Login to comment';
+    document.querySelector('#comments-form').classList.add('hide');
```

Comment:
since fetching takes some time, it is better to set hide class by default, and remove it if user is logged in.

Also, the for loginLink, its better to change it to some kind of loading

Reply:
Hi Kevin, not sure what you mean by loading?